stories/ui/Badge.js

```diff
@@ -1,21 +1,55 @@
 /* @flow */
 import React from 'react';
 import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
 
+// $FLOW_FIXME flow not liking the path here ( will look into this more)
```

Comment:
You will need to add a [resolver](https://github.com/mozilla/addons-frontend/blob/master/.flowconfig#L22-L23) for Flow too but I think that's a bit risky (I explained why in the comment up above).

Reply:
yep I'll do relative paths for now.